docs/api/apiv3/components/schemas/storage_model.yml

```diff
@@ -0,0 +1,59 @@
+# Schema: StorageModel
+---
+type: object
+required:
+  - id
+  - _type
+  - href
+  - storageType
+properties:
+  id:
+    type: integer
+    description: Storage id
+  _type:
+    type: string
+    enum:
+      - Storage
+  href:
+    type: string
+    format: uri
+    description: web uri of the storage instance
+  storageType:
+    type: string
+    enum:
+      - nextcloud
+    description: |-
+      The type of the storage resource. Currently only nextcloud storages are supported.
+  createdAt:
+    type: string
+    format: date-time
+    description: Time of creation
+  updatedAt:
+    type: string
+    format: date-time
+    description: Time of the most recent change to the storage
+  _links:
+    type: object
+    required:
+      - self
+    properties:
+      self:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              This storage resource
+
+              **Resource**: Storage
+example:
+  id: 1337
+  _type: Storage
+  href: https://nextcloud.deathstar.rocks/
+  storageType: nextcloud
+  createdAt: '2021-12-20T13:37:00.211Z'
+  updatedAt: '2021-12-20T13:37:00.211Z'
+  _links:
+    self:
+      href: /api/v3/storages/1337
+      title: storage
```

Comment:
A client defined title/name might be helpful.

Reply:
With our projects here in self link title is the `Name` of the project. I'd go the same approach with the storage resource, as the user/client defines that name.

Reply:
But, alas, I noticed, I forgot the storage name in the model. Added.